app/models/legislator.rb

```diff
@@ -61,7 +61,7 @@ class Legislator < ActiveRecord::Base
   scope :current_supporters,-> { joins(sponsorships: :bill).where('sponsorships.id IS NOT NULL').distinct.merge(Bill.current) }
   scope :allowed_states,-> {
     joins({district: :state}, :state)
-    .where('state.abbrev IS IN (:state_abbrev)', state_abbrev: ALLOWED_STATES) }
+    .where('states.abbrev IN (:state_abbrev)', state_abbrev: ALLOWED_STATES) }
```

Comment:
I was messing around trying to fix this scope, but it's still gonna need more work.

`joins({district: :state}, :state)` won't work -- I think it needs to have an `OR` in the join clause (like `District#legislators`)